evaluation/toolqa/README.md

```diff
@@ -0,0 +1,45 @@
+# ToolQA Evaluation with OpenDevin
+
+This folder contains evaluation harness we built on top of the original [ToolQA](https://github.com/night-chen/ToolQA) ([paper](https://arxiv.org/pdf/2306.13304)). 
+
+## Setup Environment
+
+Please follow [this document](https://github.com/OpenDevin/OpenDevin/blob/main/Development.md) to setup local develop environment for OpenDevin.
```

Comment:
typo: development

Reply:
WOLFRAM_APPID is used in the calculator tool of toolqa.